painters/data_provider.py

```diff
@@ -11,7 +11,7 @@
 
 from utils import read_lines, load_img_arr
 
-DATA_DIR = join(dirname(dirname(__file__)), 'data_small')
+DATA_DIR = '/home/chris/painters/data_all'
```

Comment:
We don't have to do this in this PR, but you should leave a TODO to clean this up into a experiment configuration parameter instead of hard coding the location.